pkgs/by-name/ne/netbird-dashboard/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  inter,
+}:
+
+buildNpmPackage rec {
+  pname = "netbird-dashboard";
+  version = "2.2.0";
+
+  src = fetchFromGitHub {
+    owner = "netbirdio";
+    repo = "dashboard";
+    rev = "v${version}";
+    hash = "sha256-GJ2G/7OO3YpEGwu/S87rKEHxwVq3upBfBx09uQHQce0=";
+  };
+
+  npmDepsHash = "sha256-ts3UuThIMf+wwSr3DpZ+k1i9RnHi/ltvhD/7lomVxQk=";
+  npmFlags = [ "--legacy-peer-deps" ];
+
+  patches = [ ./01-inter.patch ];
```

Comment:
Can we have an issue for this patch to upstream and explain our needs?

Reply:
I opened an issue upstream https://github.com/netbirdio/dashboard/issues/375

Reply:
Great news, netbird vendored the font in the latest release !